docs/Multisig.md

```diff
@@ -0,0 +1,138 @@
+# MultiSig support in BTCPay Server
+
+BTCPay Server v2.1.0 introduced refined support for multi signature (multisig) wallets designed to empower your setup with added security, collaboration, and flexibility.
+
+This document provides step-by-step instructions for configuring and utilizing multisig wallets within your BTCPay Server instance.
+
```

Comment:
We need to embed a video here.

Reply:
622059f81f82f8bde468039adf3be05c317196a1